### ISSUE TITLE
PowerTag topic and ptTopic

### ISSUE THREAD

**Issue by albydnc** [albydnc/node-red-contrib-schneider-powertag#4]
@hotswapster I've seen your extractor.js code, thanks for contributing! As I read, I've seen you are using msg.ptTopic to pass info to the extractor, meanwhile I was using the msg.topic in the powertag.js, I think we should decide what convention to utilize. My choice was based on the fact that NodeRED has topic as a core object of the message. Let me know your opinion!

**Comment by hotswapster** [comment]
@albydnc 

Yeah it was a first pass to get something moving. I took that code from my flow. I have used that terminology but in all honesty, it was me learning how the messages work and I kept fudging it until I got it functional. I think we need to build a table for reference with the object and it's properties. 

The way I'd like to use this node, is in a flow where a config is loaded near the beginning of the flow and have all of the properties flow through to be used in the different areas. This means they'd need to pass through the powerTag and extractor nodes. We need to think about the application these nodes will be used in.

I've attached a rough draft of a list. It's a brain dump and can be changed in any way. Is there a proper way to handle this kind of information for reference? If we get this part correct at the start, it will be easier later.

[powerTag_lexicon.xlsx](https://github.com/albydnc/node-red-contrib-schneider-powertag/files/3389796/powerTag_lexicon.xlsx)

**Comment by hotswapster** [comment]
Or maybe some of that information should be stored as global context?
